admin_manual/installation/nginx_examples.rst

```diff
@@ -114,6 +114,13 @@ your nginx installation.
       error_page 403 /core/templates/403.php;
       error_page 404 /core/templates/404.php;
   
+      # Fixes Windows WebDav client error 0x80070043 "The network name cannot be found."
+      location = / {
+          if ($request_method = OPTIONS) {
+              return 301 $scheme://$server_name/remote.php/webdav/;
```

Comment:
Actually, in this situation it does not matter - we just should redirect `DavClnt` somwhere, where server returns `401 Unauthorized` code... Hmm.. and what if we will not send `301 Redirect`, but `401 Unauthorized` at once?
I just test it out and it works! Think it is better solution